nnvm/python/nnvm/testing/tf.py

```diff
@@ -153,6 +153,35 @@ def read_normalized_tensor_from_image_file(file_name,
     np_array = normalized.eval()
     return np_array
 
+def get_workload(model_path):
+    """ Import workload from frozen protobuf
+
+    Parameters
+    ----------
+    model_path: str
+        model_path on remote repository to download from.
+
+    Returns
+    -------
+    graph_def: graphdef
+        graph_def is the tensorflow workload for mobilenet.
+
+    """
+
+    repo_base = 'https://github.com/dmlc/web-data/raw/master/tensorflow/models/'
+    model_name = os.path.basename(model_path)
+    model_url = os.path.join(repo_base, model_path)
+
+    from mxnet.gluon.utils import download
+    download(model_url, model_name)
+
+    # Creates graph from saved graph_def.pb.
+    with tf.gfile.FastGFile(os.path.join("./", model_name), 'rb') as f:
+        graph_def = tf.GraphDef()
+        graph_def.ParseFromString(f.read())
+        graph = tf.import_graph_def(graph_def, name='')
+        return graph_def
+
 def get_workload_inception_v3():
```

Comment:
get_workload_inception_v3() and get_workload_inception_v1() should also have only one line code as get_workload_mobilenet()
No need to prepare inpur data here, use random input for all tests

Reply:
InceptionV1 and V3 has some challenges using random data now.

- Pool operators has difference in implementation with SAME padding. we use pad operator to meet 4 sides padding but the pad_value is 0 or passed value. In fact we should not consider the padded value in operation (avg/ max). At least maxpool I could workaround by pad_value = -(FLOAT_MAX) but avg pool is a tricky now.

- Actual input for V1 is jpeg encoded stream. TF use decode_jpeg operator for this. NNVM we bypass it and feed decoded frame directly. I could try this by taking random_data, encoding and giving to tensorflow once the pool operators perform consistently.

I will revisit these test_cases once I get the pool ops working in sync with tensorflow.

Reply:
I see. Since now count_include_pad is introduced to AvgPool: https://github.com/dmlc/nnvm/pull/498, can you use it to handle same padding?

Reply:
Great. In fact the pool operator test_cases I kept on hold on these reasons.
I will revisit the pool ops now. Thanks.

Reply:
@Huyuwei 

Taking this discussion to https://discuss.tvm.ai/t/avgpool-same-padding/314 to resolve avg_pool issue.